src/App.vue

```diff
@@ -7,21 +7,29 @@
 <script lang="ts">
```

Comment:
This looks like the old version of develop. I don't see the changes in here from my latest merge.
Can you please pull develop and rebase that into this branch and then re-push
@Ataww

Reply:
Will do